src/test/java/com/opentable/extension/BodyTransformerTest.java

```diff
@@ -70,6 +70,58 @@ public void replaceNestedVariablesForXml() throws Exception {
 		testNestedField(requestBody);
 	}
 
+    @Test
+    public void replaceVariableHolderFromKeyValueRequest() throws Exception {
+        final String requestBody = "utf8=%E2%9C%93&auth_cv_result=M&req_locale=en-us&decision_case_priority=3";
+        testKeyValueBodyRequest(requestBody);
+
+    }
+
+	@Test
+	public void replaceVariableHolderFromKeyRequest() throws Exception {
+		final String requestBody = "EmptyKey=";
+		testKeyBodyRequest(requestBody);
+	}
+
+    private void testKeyValueBodyRequest(String body) {
+        wireMockRule.stubFor(post(urlEqualTo("/get/this"))
+            .willReturn(aResponse()
+                .withStatus(200)
+                .withHeader("content-type", "application/json")
+                .withBody("{\"auth_cv_result\":\"$(auth_cv_result)\"}")
+                .withTransformers("body-transformer")));
+        given()
+            .contentType("application/x-www-form-urlencoded")
+            .body(body)
+        .when()
+            .post("/get/this")
+        .then()
+            .statusCode(200)
+            .body("auth_cv_result", equalTo("M"));
+
+        wireMockRule.verify(postRequestedFor(urlEqualTo("/get/this")));
+
+    }
+	private void testKeyBodyRequest(String body) {
+		wireMockRule.stubFor(post(urlEqualTo("/get/this"))
+			.willReturn(aResponse()
+				.withStatus(200)
+				.withHeader("content-type", "application/json")
+				.withBody("{\"EmptyKey\":\"\"}")
```

Comment:
If this is returning empty string, then we'd never know that it tried to parse the "EmptyKey" url body?